airbyte-integrations/connectors/source-greenhouse/acceptance-test-config.yml

```diff
@@ -4,10 +4,12 @@ connector_image: airbyte/source-greenhouse:dev
 tests:
   spec:
     - spec_path: "source_greenhouse/spec.json"
+      backward_compatibility_tests_config:
+        disable_for_version: "0.2.7"
   connection:
     - config_path: "secrets/config.json"
       status: "succeed"
-    - config_path: "secrets/config_users_only.json"
```

Comment:
these credentials don't work, but I confirmed there are `users` records in the account configured in `"secrets/config.json"`